clients/p4rt-ctl/p4rt-ctl.in

```diff
@@ -1760,24 +1767,37 @@ def validate_args(argv, command, expected_nr):
         raise Exception("p4rt-ctl: '{}' command requires at least {} "
                         "arguments".format(command, expected_nr))
 
+grpc_server_addr = ""
```

Comment:
Recommend moving this to top of file

Reply:
Should default be set to `localhost`?

Reply:
Prefer keeping global variable just above main where it's being used . I can add a comment if that makes it better.